pkg/static/login.js

```diff
@@ -622,7 +622,7 @@ import "./login.scss";
 
             const b1 = document.createElement("button");
             b1.textContent = host;
-            b1.classList.add("pf-c-button", "pf-m-tertiary", "host-name");
+            b1.classList.add("pf-v5-c-button", "pf-m-tertiary", "host-name");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18806-20230519-134728-54b4f0d5-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)